crates/db_views/src/post_view.rs

```diff
@@ -426,8 +423,7 @@ pub struct PostQuery<'a> {
   pub search_term: Option<String>,
   pub url_search: Option<String>,
   pub saved_only: Option<bool>,
-  pub moderator_view: Option<bool>,
-  pub is_profile_view: bool,
+  pub is_profile_view: Option<bool>,
```

Comment:
I dont see any reason to change this to option. In fact there is another PR open now to get rid of most options.

Reply:
Thanks for pointing that out, I seem to have messed up a merge commit.